e2e-tests/IIoTPlatform-E2E-Tests/Standalone/A_PublishSingleNodeStandaloneTestTheory.cs

```diff
@@ -41,6 +41,7 @@ IIoTMultipleNodesTestContext context
         async Task SubscribeUnsubscribeTest(MessagingMode messagingMode) {
             var ioTHubEdgeBaseDeployment = new IoTHubEdgeBaseDeployment(_context);
             var ioTHubPublisherDeployment = new IoTHubPublisherDeployment(_context, messagingMode);
+            var ioTHubLegacyPublisherDeployment = new IoTHubLegacyPublisherDeployments(_context);
```

Comment:
This specific test suite is not using legacy publisher and `ioTHubLegacyPublisherDeployment` variable is not used later on. Can we remove it from here ?

Reply:
removed